source/_integrations/home_connect.markdown

```diff
@@ -44,6 +44,9 @@ Note that it depends on the appliance and on API permissions which of the featur
 - Redirect URI: `https://my.home-assistant.io/redirect/oauth`
 - Go to `https://my.home-assistant.io/` and make sure that your Home Assistant URL is set there. For example: `http://homeassistant:8123/` or `http://homeassistant.local:8123`
 
+4. On success, you will be redirect to the Applicatinos page. Click on "Details" for your app. Make note of the client ID and secret - you will need it for the next step. Log out of the Home Connect developer portal.
```

Comment:
The numbering is correct because there are other list items. I will apply the other suggestions though if you update the suggestion to exclude the numbering change